### PR TITLE
Regenerate release notes with AMDMIGraphX

### PR DESCRIPTION
**Pull request by samjwu**

### CODE REVIEW

**Comment by saadrahim** [comment]
develop or roc-5.7.x as the target branch?

**Comment by samjwu** [comment]
> develop or roc-5.7.x as the target branch?

The branches have diverged enough that changing the target of this PR will include other changes from develop. I have created https://github.com/RadeonOpenCompute/ROCm/pull/2544 specifically to include these changes in the roc-5.7.x branch.

File: RELEASE.md [hunk 1]
```diff
@@ -19,15 +19,445 @@ The release notes for the ROCm platform.
...
+#### AMDMIGraphX 2.7
+
+MIGraphX 2.7 for ROCm 5.7.0
+
+##### Added
+
+- Enabled hipRTC to not require dev packages for migraphx runtime and allow the ROCm install to be in a different directory than it was during build time
+- Add support for multi-target execution
+- Added Dynamic Batch support with C++/Python APIs
+- Add migraphx.create_argument to python API
```
Comment (Rmalavally) [inline RELEASE.md:207]: @samjwu I didn't get around to making the minor changes to the MIGraphX section. Please correct all instances of Add to Added.

File: RELEASE.md [hunk 2]
```diff
@@ -19,15 +19,445 @@ The release notes for the ROCm platform.
...
+- Added dockerfile example for Ubuntu 22.04
+- Add TensorFlow supported ops in driver similar to exist onnx operator list
+- Add a MIGRAPHX_TRACE_MATCHES_FOR env variable to filter the matcher trace
+- Improved debugging by printing max,min,mean and stddev values for TRACE_EVAL = 2
+- use fast_math flag instead of ENV flag for GELU
+- Print message from driver if offload copy is set for compiled program
+
+##### Optimizations
```
Comment (Rmalavally) [inline RELEASE.md:215]: Likewise:

Change to section to:


- Optimized for ONNX Runtime 1.14.0
- Improved compile times by only building for the GPU on the system
- Improved performance of pointwise/reduction kernels when using NHWC layouts
- Loaded specific version of the migraphx_py library
- Annotated functions with the block size so the compiler can do a better job of optimizing 
- Enabled reshape on nonstandard shapes
- Used half HIP APIs to compute max and min
- Added support for broadcasted scalars to unsqueeze operator
- Improved multiplies with dot operator
- Handled broadcasts across dot and concat
- Added verify namespace for better symbol resolution

File: RELEASE.md [hunk 3]
```diff
@@ -19,15 +19,445 @@ The release notes for the ROCm platform.
...
+- Load specific version of the migraphx_py library
+- Annotate functions with the block size so the compiler can do a better job of optimizing 
+- Enable reshape on nonstandard shapes
+- Use half HIP APIs to compute max and min
+- Added support for broadcasted scalars to unsqueeze operator
+- Improved multiplies with dot operator
+- Handle broadcasts across dot and concat
+- Add verify namespace for better symbol resolution
+
+##### Fixed
```
Comment (Rmalavally) [inline RELEASE.md:229]: Corrected version:


- Resolved accuracy issues with FP16 resnet50
- Updated cpp generator to handle inf from  float
- Fixed assertion error during verify and make DCE work with tuples
- Fixed convert operation for NaNs
- Fixed shape typo in API test
- Fixed compile warnings for shadowing variable names
- Added missing specialization for the `nullptr` for the hash function

File: RELEASE.md [hunk 4]
```diff
@@ -19,15 +19,445 @@ The release notes for the ROCm platform.
...
+
+- Resolved accuracy issues with FP16 resnet50
```
Comment (Rmalavally) [inline RELEASE.md:231]: Corrected version


- Resolved accuracy issues with FP16 resnet50
- Update cpp generator to handle inf from  float
- Fix assertion error during verify and make DCE work with tuples
- Fix convert operation for NaNs
- Fix shape typo in API test
- Fix compile warnings for shadowing variable names
- Add missing specialization for the `nullptr` for the hash function

##### Changed

- Bumped version of half library to 5.6.0
- Bumped CI to support ROCm 5.6
- Make building tests optional
- Replaced np.bool with bool as per numpy request

File: RELEASE.md (hunk 1 above)
Comment (samjwu) [reply RELEASE.md:207]: These changes are automatically pulled from https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/develop/CHANGELOG.md

We need to make the changes there; anything changed here will be overwritten when the script is run again.

Comment (Rmalavally) [reply RELEASE.md:207]: @samjwu Makes sense. Let me reach out to Chris Austen.

**Review by Rmalavally — APPROVED** [review]

**Pull request merged**